utils/NotificationClient.js

```diff
@@ -18,7 +19,7 @@ module.exports = (ticketId, subject, content, emailIds, requester) => {
     /**
      * We can keep this hardcoded URL in the configs files
      */
-    client.post("http://localhost:7777/notifiServ/api/v1/notifications", args, function (data, response) {
+    client.post(`http://localhost:${serverConfig.PORT}/notifiServ/api/v1/notifications`, args, function (data, response) {
```

Comment:
How is this change preventing the crash ? We are just replacing 7777 with a value from config file. Am I missing anything ?